Taxonomy and metadata naming convension (#119)
MrLongbottom/P10-Article: sections/discussion/taxonomy-topic.tex

\subsection{Taxonomy-topic model}\label{sec:taxonomy_analysis}
Finally, we also want to analyze the taxonomy-topic model, especially since this model has the highest topic coherence out of all the tested models.
\autoref{tab:pachinko_selected_topics} shows selected lowest level topics from the the taxonomy-topic model.

> As can be seen from the figure most of the topics are quite good, However there are some topics (e.g., topic 19 and 42) that consist entirely of words that provide little context or semantic meaning.

Reviewer: ```suggestion
As can be seen from the figure most of the topics are quite good. 
However, there are some topics (e.g., topics 19 and 42) that consist entirely of words that provide little context or semantic meaning.
```

\end{figure}

\autoref{tab:pachinko_mid_topics} gives an overview of how the taxonomy topics in the third layer of the taxonomy-topic model, are connected to the fourth layer topics that were generated by the model.

> Some of these connections make a lot of sense, such as the 'Økonomi' (Economy) taxonomy entry topic which has the three filler topics (ie. topic which consists entirely of words with little semantic value) within the top 5 most probable topics: 79, 75, and 42, as well as two topics which are about money: 74 and 9. \autoref{tab:pachinko_selected_topics} shows the top words for each of these topics.

Reviewer 2: ```suggestion
Some of these connections make a lot of sense, such as the 'Økonomi' (Economy) taxonomy entry topic which has the three filler topics (i.e., topics which consists entirely of words with little semantic value) within the top 5 most probable topics: 79, 75, and 42, as well as two topics which are about money: 74 and 9.
\autoref{tab:pachinko_selected_topics} shows the top words for each of these topics.
```

However not all the connections between higher and lower level topics are as understandable as these. 

> For example, the 'Kriminalitet' (Crime) taxonomy entry topic has two filler topics within the 5 most probable topics: 42 and 75, one topic about economy: 60, one topic about politics: 8, and one topic about sports: 86.

Reviewer 2: ```suggestion
For example, the 'Kriminalitet' (Crime) taxonomy entry has two filler topics within the 5 most probable topics: 42 and 75, one topic about economy: 60, one topic about politics: 8, and one topic about sports: 86.
```
